### PR TITLE
Update positions

### PR DESCRIPTION
**Pull request by LeonVreling**

### CODE REVIEW

**Review by WikiRik — APPROVED** [review]

File: models/Position.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const Position = sequelize.define('position', {
     start_term: {
         type: Sequelize.DATEONLY,
         allowNull: false,
-        defaultValue: Sequelize.NOW,
```
Comment (serge1peshcoff) [inline models/Position.js:38]: shouldn't it be `Sequelize.fn('NOW')` as well?

**Comment by LeonVreling** [comment]
The default value is now an empty string for the start of the term, so the same as the force close of the application. Furthermore, the migration is updated to work correctly, like with the addition of the force close field

File: models/Position.js (hunk 1 above)
Comment (LeonVreling) [reply models/Position.js:38]: That is no longer a thing, it is now an empty string (like force_ends)

Comment (serge1peshcoff) [reply models/Position.js:38]: it won't pass the validation then right?

**Comment by WikiRik** [comment]
Fuck, forgot to change the name before I merged ;(

File: models/Position.js (hunk 1 above)
Comment (LeonVreling) [reply models/Position.js:38]: What validation, there is none yet. Rik made a separate issue for that to make JC happy now

**Pull request merged**